sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosEndToEndOperationLatencyPolicyConfigBuilder.java

```diff
@@ -45,4 +48,26 @@ public CosmosEndToEndOperationLatencyPolicyConfigBuilder enable(
         return this;
     }
 
+    /**
+     * Sets the availability strategy to be used for the policy.
+     *
+     * @param availabilityStrategy the availability strategy to be used for the policy
+     * @return current CosmosEndToEndOperationConfigBuilder
+     */
+    public CosmosEndToEndOperationLatencyPolicyConfigBuilder availabilityStrategy(
+        AvailabilityStrategy availabilityStrategy) {
+        this.availabilityStrategy = availabilityStrategy;
+        return this;
+    }
+
+    /**
+     * Sets the feed operation timeout. Defaults to endToEndOperationTimeout. Use this if you need to set a different timeout
+     * for feed operations
+     * @param endToEndNonPointOperationTimeout feed operatoin timeout Duration
+     * @return current CosmosEndToEndOperationConfigBuilder
+     */
+    public CosmosEndToEndOperationLatencyPolicyConfigBuilder endToEndNonPointOperationTimeout(Duration endToEndNonPointOperationTimeout) {
+        this.endToEndNonPointOperationTimeout = endToEndNonPointOperationTimeout;
```

Comment:
Do we need some checks here - null check / negative values / other boundary conditions?

Reply:
I recall @mbhaskar  mentioning about some validations on build or actual config, if not, we should have some validations for sure.

Reply:
Also, if there are any validations we are thinking about, worth adding them now, to avoid any behavior breaking changes in the future.

Reply:
Done